components/concept/storage/src/main/java/mozilla/components/concept/storage/CreditCardsAddressesStorage.kt

```diff
@@ -214,6 +215,44 @@ data class CreditCard(
     }
 }
 
+/**
+ * Credit card autofill entry.
+ *
+ * This contains the data needed to handle autofill but not the data related to the DB record.
```

Comment:
seems like it's both though? e.g. there's an optional `guid` field ... It's pretty confusing that it's "both". If we can remove `guid` from this, we'll have a clean separation - `CreditCard` is used on the way "out", and `CreditCardEntry` is used on the way "in".

Is that the correct interpretation?

Reply:
`CreditCardEntry` is a wrapper around the GV Credit Card entry is the best interpretation. We can't remove the `guid` because when we autocomplete a credit card entry to the form and update the credit card, GV will also return a save credit card prompt with provided `guid` so we can find the credit card to update.